test/junit/src/test/scala/strawman/collection/immutable/TreeSetTest.scala

```diff
@@ -17,4 +17,60 @@ class TreeSetTest {
     assertEquals(set, set drop Int.MinValue)
```

Comment:
I don’t think we need these tests since `TreeSet` inherits the default implementation, which is already tested in `IterableViewLikeTest`.

Reply:
`TreeSet` has overloaded `zip` and `zipWith` methods inherited from `SortedSet`. These tests are testing the overloaded methods.

Reply:
Oh yes, that’s right.